code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -18,22 +18,91 @@
 	display_name = "a pack of Midoris"
 	path = /obj/item/storage/fancy/cigarettes/cigpack_midori
 
+/datum/gear/marijuanapack
+	display_name = "a pack of medical Marijuana"
+	path = /obj/item/storage/fancy/cigarettes/cigpack_med
+
+/datum/gear/cigar
+	display_name = "Havana Cigar"
+	path = /obj/item/clothing/mask/cigarette/cigar/havana
+	cost = 2
+
+/datum/gear/smokingpipe
+	display_name = "smoking pipe"
+	path = /obj/item/clothing/mask/cigarette/pipe
+	cost = 2
+
 /datum/gear/lighter
 	display_name = "a cheap lighter"
 	path = /obj/item/lighter
 
+/datum/gear/matches
+	display_name = "a box of matches"
+	path = /obj/item/storage/box/matches
+
+/datum/gear/zippo
+	display_name = "zippo lighter"
+	path = /obj/item/lighter/zippo
+	cost = 2
```

Comment:
Premium item only available in cig vendors--highly skeptical.

Reply:
Same, remove as per Fox